### DISCUSSION TITLE
Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:3.0.1:sig

### DISCUSSION THREAD

**Discussion by lansgg** [apache/skywalking#10626] [Ideas]
I want to mvn install ,
but  error:
"
Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:3.0.1:sign (sign-release-artifacts) on project apm: Execution sign-release-artifacts of goal org.apache.maven.plugins:maven-gpg-plugin:3.0.1:sign failed.
"

I will what to do

pls

**Comment by wu-sheng** [comment]
This is not about SkyWalking. Search for maven gpg-plugin instead.
